backend/app/anti_recommenders/anti_recommender_proxy.py

```diff
@@ -0,0 +1,32 @@
+import os
+from typing import Tuple 
+from .anti_recommender import AntiRecommender
+from .open_ai.regular_open_ai_anti_recommender import RegularOpenAIAntiRecommender
+
+class AntiRecommenderProxy(AntiRecommender):
+    """
+    The Proxy class for the anti-recommendation engine.
+    """
+
+    def __init__(self, wikipedia_title: str = None):
+        self.wikipedia_title : str = wikipedia_title
+        self.anti_recommender = None
+
+    def set_title(self, title:str) -> None:
```

Comment:
Either just have a public property `self.wikipedia_title` or (preferable) use property getters (`@property def title(self)`: / setters (`@title.setter def set_title(self, title)`) and a private member (`self.__wikipedia_title`).

Reply:
Yes, I am using the property getters/setters now.